simple_token_authentication.gemspec

```diff
@@ -16,9 +16,11 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,doc,lib}/**/*", "CHANGELOG.md", "LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*", "gemfiles/*.gemfile", "gemfiles/*.gemfile.lock", "Appraisals"]
 
-  s.add_dependency "actionmailer", ">= 3.2.6", "< 7"
-  s.add_dependency "actionpack", ">= 3.2.6", "< 7"
+  s.add_dependency "actionmailer", ">= 3.2.6", "< 8"
+  s.add_dependency "actionpack", ">= 3.2.6", "< 8"
   s.add_dependency "devise", ">= 3.2", "< 6"
+  s.add_dependency "activemodel-serializers-xml"
+  s.add_dependency "active_model_serializers"
```

Comment:
**TL;DR**: There is one more version constraint to update:

```diff
- s.add_development_dependency "activerecord", ">= 3.2.6", "< 7"
+ s.add_development_dependency "activerecord", ">= 3.2.6", "< 8"
```

The **longer version**:

You should be able to test this branch locally, as follows:

1. Use Ruby 3.2.0 (the latest one, because that's a good default)
2. Remove the `Gemfile.lock` if you've got one, to make sure you're actually testing the new version constraints
3. `bundle install`
4. `rake` (runs the test suite by default, should be successful :green_apple:)

If you do that (you might already have done it), you'll notice that `bundle install` fails, because the constraint around `activerecord` makes no sense to `bundler`. It cannot go high enough to match the other dependencies, and as a result `bundler` tries confusingly to make things work with the oldest version it finds... and fails.

Updating the constraint fixes that. :slightly_smiling_face: 

Once that done, you'll be able to go through the test plan above, and the new automated [**Test**](https://github.com/gonzalo-bulnes/simple_token_authentication/actions/workflows/test.yml) workflow will be happy too. (The automated workflow takes care of running regression tests against older versions of Ruby, Rails and Devise so you don't have to.)